qa/suites/fs/upgrade/mds_upgrade_sequence/overrides/ignorelist_upgrade.yaml

```diff
@@ -2,3 +2,4 @@ overrides:
   ceph:
     log-ignorelist:
       - OSD_DOWN
+      - cluster *\[WRN\] *osd.*is down
```

Comment:
oh wow - regex's are allowed. That's because these strings are used in `grep`, yes?

Reply:
Yes, grep allows regex so it should be fine. There's on other way to catch that error message otherwise.

Reply:
what's wrong with `osd.*is down`?

Reply:
Current pattern ensures that regex matches no messages other than cluster warning message.

Reply:
But why do we need to be that specific? We've often had to put the same type of message in the ignorelist twice because it appears as a health warning and a cluster message.

Reply:
Okay, I didn't want to ignore unnecessary message. Will change this.

Reply:
Done - https://github.com/ceph/ceph/compare/acdbe870553f3ccc24502ab0d53dfa71261f9e8e..9ae2c89511be4b64c17974cd0fc6770641f9af4d